scst-store/additional.hbs.md

```diff
@@ -15,6 +15,7 @@ This section includes the following topics:
 - [Ingress support](ingress.hbs.md)
 - [Custom certificate configuration](custom-cert.hbs.md)
 - [TLS configuration](tls-configuration.hbs.md)
+- [Cert rotation](cert-rotation.hbs.md)
```

Comment:
Adding it to `additional.hbs.md` is good.

Also you need to add it to `toc.md`. Showing up in the table of contents is what the docs teams wants for every doc.